src/main/scala/com/exasol/cloudetl/helper/ExasolColumnValueProvider.scala

```diff
@@ -41,7 +48,13 @@ final case class ExasolColumnValueProvider(iterator: ExaIterator) extends JavaCl
     } else {
       val updatedBigDecimal = bigDecimal.setScale(scale, RoundingMode.HALF_UP)
       if (updatedBigDecimal.precision > precision) {
-        throw new IllegalArgumentException(s"Actual precision of big decimal value exceeds configured '$precision'.")
+        throw new IllegalArgumentException(
+          ExaError
+            .messageBuilder("E-CSE-24")
+            .message("Actual precision of big decimal value exceeds configured {{PRECISION}}.")
```

Comment:
You could maybe add the value to make it easier to find that record. And in addition add the actual precision.